PoshBot/Plugins/Builtin/Public/Slap.ps1

```diff
@@ -0,0 +1,23 @@
+function Slap {
+    <#
+    .SYNOPSIS
+        Slap a user with a large trout
+    .PARAMETER User
+        The user that will be slapped
+    .EXAMPLE
+        !slap --user jaap
+    #>
+
+
+    [PoshBot.BotCommand(Permissions = 'view')]

```

Comment:
No not in particular, I have run into some occasions where users without any roles are not able to run commands that do not have permissions explicitly set. But as far as I am concerned it can be removed here.